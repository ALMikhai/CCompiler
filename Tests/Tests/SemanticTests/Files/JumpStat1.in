int f() {
    for (;;) {
        for (;;) {
            continue;
        }
        break;
    }
}
