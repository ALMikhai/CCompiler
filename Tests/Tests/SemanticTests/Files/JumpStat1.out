\-TRANSLATIONUNIT
  \-FUNC
    |-DECLSPEC
    | |-INT
    |-INITDECLARATOR
    | \-DECLARATOR
    |   \-FUNC
    |     |-f
    |     \-EMPTY
    \-COMPOUNDSTAT
      \-STATLIST
        \-FOR
          |-EMPTY
          |-EMPTY
          |-EMPTY
          \-COMPOUNDSTAT
            \-STATLIST
              |-FOR
              | |-EMPTY
              | |-EMPTY
              | |-EMPTY
              | \-COMPOUNDSTAT
              |   \-STATLIST
              |     \-CONTINUE
              \-BREAK

Structs { }
Symbols {
	FUNC returning VOID
	Arguments{
		STRING :: s
		INT :: o
	}:: printf
	FUNC returning INT
	Arguments{ }
	Structs { }
	Symbols { } :: f
}
